docs/GettingStarted.rst

```diff
@@ -1348,7 +1348,8 @@ manager.
 
 4. Click ``Yes`` button in the confirmation message to accept your deletion.
 
-.. note:: As an eXo Platform user, you can only delete your own activities. If you are manager of a space, you can delete any activities related to this space.
+.. note:: As an eXo Platform user, you can only delete your own activities/comments. 
+          If you are manager of a space, you can delete any activities or comments related to this space.
```

Comment:
```suggestion
          If you are manager of a space, you can delete any activities or comments posted in this space.
```

Reply:
@mzorai @hghorbel  can an space manager edit activities/comments in his space ?

Reply:
No the space's manager can just delete activities or comments of other users on his space. He can not edit them.

Reply:
rectified in the note